webpack.config.js

```diff
@@ -44,8 +44,10 @@ var options = {
     devtools: path.join(__dirname, 'src', 'pages', 'Devtools', 'index.js'),
     panel: path.join(__dirname, 'src', 'pages', 'Panel', 'index.jsx'),
   },
-  chromeExtensionBoilerplate: {
-    notHotReload: ['background', 'contentScript', 'devtools'],
+  custom: {
+    notHMR: ['background', 'contentScript', 'devtools'],
+    enableBackgroundAutoReload: true, // always true when "enableContentScriptsAutoReload" is set true
```

Comment:
It might be nice to link somewhere in the documentation that explains this reasoning.